### PR TITLE
Break up sdk header into language and version

### PR DESCRIPTION
**Pull request by bpross**
https://procoretech.atlassian.net/browse/ZL-1055

Break up `Procore-Sdk-Version` to send language in `Procore-Sdk-Language`

### CODE REVIEW

**Review by njbbaer — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,10 @@
-## Unreleased
+# Unreleased
```
Comment (gilmcquillan) [inline CHANGELOG.md:1]: Why this change?

**Review by njbbaer — APPROVED** [review]

**Review by gilmcquillan — APPROVED** [review]
